webpack.config.js

```diff
@@ -28,6 +28,7 @@ module.exports = {
             loader: 'babel-loader',
             options: {
               cacheDirectory: true,
+              plugins: ['@babel/plugin-transform-object-assign'],
```

Comment:
This re-implements a transformer specific to `Object.assign`. `@babel/env` does not support library-level transforms for this call without injecting a polyfill that pollutes the global namespace.